### ISSUE TITLE
register_alias_force() breaks get_content_id()

### ISSUE THREAD

**Issue by Treer** [minetest/minetest#9632]
##### Minetest version

5.2.0 Release, but PR #9458 brought the problem to light

##### Summary
`minetest.register_alias_force(a, b)` can cause `get_content_id(a)` to fail.

before 5.2.0 it failed silently, now it causes the game to exit, but it shouldn't be failing at all - the lookup should succeed.

Mods that use get_content_id() won't know whether it will fail/exit before they invoke it because *another mod* may have invoked register_alias_force(). Having mods always perform an alias lookup before every get_content_id() call works around this, but doesn't feel like the right solution.

An example is the nether mod, which has a dependency on stairs. If another mod (moreblocks) invokes register_alias_force("stairs:stair_cobble", "moreblocks:stair_cobble") then nether will crash when it tries to get the id of a node it has a dependency for.

I've discussed it further in PR #9458

##### Steps to reproduce

It doesn't always seem to reproduce, so here's a mod that does it
[aliastest.zip](https://github.com/minetest/minetest/files/4464137/aliastest.zip)

Create a new MTG world with just that mod and run it.

The mod runs the following code with a dependency on default
```
-- in the beginning, get_content_id() works
local it_works  = minetest.get_content_id("default:cobble")
local it_works2 = minetest.get_content_id("default:obsidian")
local it_works3 = minetest.get_content_id("default:dirt_with_grass")

-- register_alias() doesn't stop get_content_id() from working
minetest.register_alias("default:obsidian", "default:dirt_with_grass")
local it_still_works = minetest.get_content_id("default:obsidian")

-- register_alias_force() breaks get_content_id()
minetest.register_alias_force("default:cobble", "default:dirt_with_grass")
local it_fails = minetest.get_content_id("default:cobble")


minetest.log("before alias, id=" .. it_works)
minetest.log("after alias, id=" .. it_still_works)
minetest.log("after alias-force, id=" .. it_fails)
```

**Comment by grantcarthew** [comment]
Hey,
I just spent the morning debugging this in my server.
It looked like the `underch` (Under Challange) mod was the cause, however the source of the issue was in moreblocks.

Here is the line causing the problem: https://github.com/minetest-mods/moreblocks/blob/6f9b787f3e2badbb4646c19b37db9ba4b0546cf8/stairsplus/registrations.lua#L74

Was scratching my head until I found this issue. Thanks guys. Looking forward to a fix.
